lib/performance_platform.rb

```diff
@@ -0,0 +1,65 @@
+module PerformancePlatform
+  PP_DOMAIN = 'https://www.performance.service.gov.uk'.freeze
+
+  class Base
+    attr_reader :headers
+
+    def initialize(token = nil)
+      token || abort('No token set. Note that this task should only be executed in production.')
```

Comment:
If not running in production is important enough to write this error perhaps it also justifies a guard clause that checks for `Rails.env.production?`

Reply:
I don't want to guard this as I intend to use it in staging first, and then request for any submitted information to be reset so we can start using it in production.

As abort raises an exception any issues with the the token not being set should be identifiable via rollbar errors.